install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,118 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
```

Comment:
s/cluster which/cluster, which

Reply:
Could "lots of shard rebalancing" be written as "shard rebalancing"? Or is it the amount that is problematic?

Reply:
I believe its mostly the volume that is the problem. A cluster could have rebalancing happen from normal operation, its the excessive rebalancing that happens when a cluster disappears that we want to avoid because it would rebalance due to its absence and the rebalance again when its back.